### PR TITLE
[EA Forum only] Small fixes for Wrapped and a couple other things

### PR DESCRIPTION
**Pull request by s-cheng**
Various small fixes, links to slack conversations:

1. [Remove scroll snapping from Wrapped](https://cea-core.slack.com/archives/C0680RXPL78/p1704195221474609)
2. Some cleanup around the Wrapped notifications
3. [Small updates to the Profile page](https://cea-core.slack.com/archives/C03R2G12W2K/p1704159211913879)
4. While I was there, also fixed the issue where the "Participation" tab can appear even if it's empty
5. [Add tracking for DMing a user via finding them in search autocomplete](https://cea-core.slack.com/archives/C03G0AQ925P/p1704158042160959)
6. [Added an ad for Wrapped in the home page RHS:](https://cea-core.slack.com/archives/C0680RXPL78/p1704195087560509)

Screen Shot 2024-01-02 at 3 05 54 PM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206267635271657) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/ea-forum/wrapped/EAForumWrapped2023Page.tsx [hunk 1]
```diff
@@ -76,16 +73,12 @@ const styles = (theme: ThemeType) => ({
   '@keyframes section-scroll-animation': {
     '0%': {
       opacity: 0,
-      // TODO: this messes with scroll snapping, see if we can add it back in
-      // transform: 'scale(0.5)'
```
Comment (s-cheng) [inline packages/lesswrong/components/ea-forum/wrapped/EAForumWrapped2023Page.tsx:80]: I played around with this but I didn't like it without the scroll snapping, so I just gave up on it

File: packages/lesswrong/server/scripts/sendWrappedNotifications.ts [hunk 2]
```diff
@@ -12,14 +11,16 @@ const sendWrappedNotifications = async () => {
...
   }, {projection: {userId: 1}}).fetch()
   
   const userIds = uniq(readStatuses.map(rs => rs.userId))
-  // filter out users who are banned
+  // filter out users who are banned, deleted, or have unsubscribed from all emails
   const users = await Users.find({
     _id: {$in: userIds},
-    banned: {$exists: false}
+    banned: {$exists: false},
+    unsubscribeFromAll: {$ne: true},
```
Comment (s-cheng) [inline packages/lesswrong/server/scripts/sendWrappedNotifications.ts:22]: I think we are not consistently blocking emailing users who unsubscribed from all, so I'm excluding them out of caution

**Comment by agnesstenlund** [comment]
Looks great! Thank you 😍  (/wrapped isn't loading, assuming that's just cause it's not on prod)

For the RHS ad–could we do a boxshadow as hover effect instead of color changes? Mostly because the background color change makes the little bulb image stand out :)

File: packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx [hunk 3]
```diff
@@ -73,6 +73,32 @@ const styles = (theme: ThemeType) => ({
...
   },
+  wrappedAd: {
+    backgroundColor: theme.palette.wrapped.background,
+    color: theme.palette.text.alwaysWhite,
+    padding: '12px 24px',
+    borderRadius: theme.borderRadius.default,
+    '&:hover': {
+      opacity: 1,
+      backgroundColor: `color-mix(in oklab, ${theme.palette.wrapped.background} 85%, ${theme.palette.text.alwaysBlack})`,
+      color: `color-mix(in oklab, ${theme.palette.text.alwaysWhite} 90%, ${theme.palette.text.alwaysBlack})`,
```
Comment (oetherington) [inline packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx:84]: I guess this is going to change based on Agnes' comment, but NB for the future, [browser support for color-mix](https://caniuse.com/?search=color-mix) is pretty bad ATM (I usually try to keep to at least 95%, and it's only 85%)

File: packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx [hunk 4]
```diff
@@ -305,33 +305,33 @@ const EAUsersProfile = ({terms, slug, classes}: {
 
   const privateSectionTabs: Array<UserProfileTabType> = [{
     id: 'drafts',
-    label: `${ownPage ? 'My' : `${username}'s`} Drafts`,
+    label: `Drafts`,
```
Comment (oetherington) [inline packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx:308]: IMO the changes in this file could have been a separate PR

**Review by oetherington — APPROVED** [review]
Code looks good to me (pending Agnes' comment) - thanks!

File: packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx (hunk 3 above)
Comment (s-cheng) [reply packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx:84]: Personally I think that's fine, it has support in all major browsers, [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix) doesn't even list the ones it doesn't support

File: packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx (hunk 4 above)
Comment (s-cheng) [reply packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx:308]: Fair

**Comment by s-cheng** [comment]
@agnesstenlund Thanks - all set! And yeah, /wrapped only works on prod due to how it pulls analytics data

**Pull request merged**